doc/examples/config/sharding.yaml

```diff
@@ -0,0 +1,39 @@
+credentials:
+  users:
+    guest:
+      roles: [super]
+
+iproto:
+  listen: 'unix/:./{{ instance_name }}.iproto'
+  advertise:
+    sharding: 'guest@'
+
+sharding:
+  bucket_count: 10000
+
+groups:
+  storages:
+    sharding:
+      roles: [storage]
+    replicasets:
+      storages-001:
+        instances:
+          storage-001:
+            database:
+              mode: rw
```

Comment:
Can we use `failover.mode: manual` in this example? I think this is better mode, because it gives more safety checks -- allows only single master. I think that the `off` mode with expicit ro/rw flags should be used if multi-master is needed.